source/main/physics/BeamFactory.h

```diff
@@ -131,7 +131,7 @@ class ActorManager
     std::unique_ptr<ThreadPool>     m_sim_thread_pool;
     std::shared_ptr<Task>           m_sim_task;
     std::vector<Actor*>             m_actors;
-    int             m_num_cpu_cores;
+    int             m_thread_pool_workers;

```

Comment:
There should be `size_t ThreadPool::GetNumWorkers()` instead

Reply:
I just realized that we don't need `m_thread_pool_workers`. I'll remove it.